pkg/base1/Makefile.am

```diff
@@ -24,6 +24,7 @@ base_DATA = \
 basedebugdir = $(debugdir)$(basedir)
 basedebug_DATA = \
 	pkg/base1/angular.js \
+	pkg/base1/angular-patternfly.css \
```

Comment:
It seems like this is being made part of the angular.js bundle below. Why do we need to add it separately here?